### DISCUSSION TITLE
4.9.1 Could not find 'fluentOverflowInitialize'

### DISCUSSION THREAD

**Discussion by stephenrichardson** [microsoft/fluentui-blazor#2405] [Q&A] [answered]
I just upgraded from 4.8.0 to 4.9.1 and am now receiving the following error. Please can you help?

![image](https://github.com/user-attachments/assets/504d0c0a-b58d-4c06-831a-504972dd55ca)

**Comment by dvoituron** [comment, marked as answer]
See #2366

You will find a solution and that will be automatic in the next release.
